.github/workflows/test.yml

```diff
@@ -8,32 +8,45 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.10", "3.11", "3.12"]  # latest release minus two
+        python-version: [ "3.9", "3.10", "3.11", "3.12"]  # latest release minus two
```

Comment:
```suggestion
        python-version: [ "3.9", "3.10", "3.11", "3.12"]
```
Might as well drop the comment if we're supporting 4 versions

Reply:
Yeah! Django 4.2 runs on Python 3.9. That's the rationale.